### PR TITLE
feat: add speakers-section.

### PR DESCRIPTION
**Pull request by eazycodinz**
I added a new feature: SpeakersSection to the project, using the images in the image folder, I also deleted the sponsor's image from the image folder since the sponsor image is been mapped from the CMS.

### CODE REVIEW

**Comment by eazycodinz** [comment]
I modified the speaker section, deleted the images from the image folder, and map the image from the CMS,

File: components/sections/SpearkersSection/SpeakersSection.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import Button from "@/components/elements/Button";
+import Image from "next/image";
+export function SpeakersSection({ blok }: { blok: SpeakersType }) {
+  console.log(blok);
```
Comment (0xJamin) [inline components/sections/SpearkersSection/SpeakersSection.tsx:5]: Remove console.log

File: types.d.ts [hunk 2]
```diff
@@ -10,6 +10,9 @@ type ElementLinkType = StoryblokBlok<"element_link"> & {
   label: string;
 };
 type StoryblokImage = {
```
Comment (0xJamin) [inline types.d.ts:12]: This is a wrong type declaration, images are not supposed to have role, image and name properties

File: components/sections/SpearkersSection/SpeakersSection.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        <div className="grid md:grid-cols-3 grid-cols-1 gap-9 justify-items-center mx-7 ">
+          {speakers.map((speakers) => (
+            <div key={speakers.image.id}>
+              <Image
+                src={speakers.image.filename}
+                alt={speakers.image.alt}
+                width={363}
+                height={432}
+              />
+              <div className="absolute bottom-5 left-2 font-semibold md:text-base text-sm">
```
Comment (0xJamin) [inline components/sections/SpearkersSection/SpeakersSection.tsx:19]: why is the position of this text set to absolute? 🤔

**Review by 0xJamin — CHANGES_REQUESTED** [review]
Screenshot 2023-08-03 at 11 38 45 AM

@eazycodinz some this is how you speaker section looks like, can we fix it?

File: types.d.ts [hunk 4]
```diff
@@ -30,6 +33,10 @@ type HeroType = StoryblokBlok<"section_hero"> & {
   description: string;
   cta_buttons: ElementLinkType[];
 };
+type SpeakersType = StoryblokBlok<"section_speakers"> & {
+  speakers: StoryblokImage[];
```
[the remark this answers is not in the document]
Comment (0xJamin) [reply types.d.ts:37]:
```
type SpeakersType = StoryblokBlok<"section_speakers"> & {
heading: string;
speakers: Speaker[];
cta_buttons: Elemmjomfodfodsf

}
```

File: types.d.ts [hunk 5]
```diff
@@ -10,6 +10,9 @@ type ElementLinkType = StoryblokBlok<"element_link"> & {
   label: string;
 };
 type StoryblokImage = {
+  name: string;
```
Comment (0xJamin) [inline types.d.ts:13]: remove this

File: components/sections/SpearkersSection/SpeakersSection.tsx (hunk 1 above)
Comment (eazycodinz) [reply components/sections/SpearkersSection/SpeakersSection.tsx]: The image text has been fixed and the console.log removed.

File: components/sections/SpearkersSection/SpeakersSection.tsx (hunk 3 above)
Comment (eazycodinz) [reply components/sections/SpearkersSection/SpeakersSection.tsx]: the absolute position have been fixed

File: types.d.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (eazycodinz) [reply types.d.ts]: the type declaration have been refactored.

File: types.d.ts (hunk 5 above)
Comment (eazycodinz) [reply types.d.ts]: it has been removed

**Comment by eazycodinz** [comment]
I refactored the type declaration, and the image text was fixed, the console.log was also removed,

**Review by 0xJamin — APPROVED** [review]
Good Job @eazycodinz , Well done🎉

**Pull request merged**